tests/parser/functions/test_concat.py

```diff
@@ -178,14 +175,3 @@ def bytes_for_len(n):
 
     a, b, c = bytes_for_len(8), bytes_for_len(0), bytes_for_len(8)
     assert contract.small_bytes4(a, b, c) == a + b + c
-
-
-def test_large_output(get_contract_with_gas_estimation, assert_compile_failed):
```

Comment:
Was this test case removed? Why?

Reply:
This has been moved to `tests/parser/syntax/test_concat.py`: https://github.com/vyperlang/vyper/blob/4b82dca5b8188b3ecb1aa85396c0d8c26e254f05/tests/parser/syntax/test_concat.py#L53-L59